wren-ai-service/README.md

```diff
@@ -1,49 +1,104 @@
 # AI Service of WrenAI
 
-## Environment Setup
+This AI service provides the Web APIs to interative with LLM for the WrenAI system. The following sections will guide 
+you through the environment setup, development, and production deployment of the AI service.
 
-- Python 3.12
-- install `poetry` with version 1.7.1: `curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`
-- execute `poetry install` to install the dependencies
-- copy `.env.example` file to `.env`, and `.env.dev.example` file to `.env.dev` and fill in the environment variables
-- [for development] execute `poetry run pre-commit install` to install the pre-commit hooks and `poetry run pre-commit run --all-files` to run the pre-commit checks at the first time to check if everything is set up correctly
-- [for development] to run the tests, execute `make test`
+## Requirements
+- Python >= 3.12
+- Docker
+- Poetry >= 1.7.1 (`curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`)
 
-## Start the service for development
+Ensure you have installed the required tools before proceeding to the next steps.
 
-- execute `make start` to start the service and go to `http://WREN_AI_SERVICE_HOST:WREN_AI_SERVICE_PORT` to see the API documentation and try the APIs
+## Setup the Development Environment
+- Begin by building the environment using the following commands:
+  ```shell
+  make setup
+  ```
+- Fill in the environment variables in the `.env` file and `.env.dev` file.
 
-## Production Environment Setup
+## Start and Stop the Service for Development
+- To run the service and the dependent services, execute the following command in the terminal and go to 
+  `http://localhost:5556` to see the API documentation and try them.
+  ```shell
+  make start
+  ```
+  - If you change the host and port in env file, you could follow the format to access the API documentation and 
+  try the Web APIs. (`http://WREN_AI_SERVICE_HOST:WREN_AI_SERVICE_PORT`)
+- To stop the services, execute the following command:
+  ```shell
+  make stop
+  ```
 
-- copy `.env.prod.example` file to `.env.prod` and fill in the environment variables
+## Run all test cases
+- All the test cases are located in the `tests` folder and the following command will run all the test cases:
```

Comment:
for make test to work, users need to start qdrant and wren-engine first, and create book_2 mdl file. you can see the command used before for reference:

```Makefile
test:
	poetry run python -m src.prepare_mdl_json --dataset_name book_2 && \
	make run-qdrant && \
	make run-wren-engine && \
	poetry run pytest -s $(args) && \
	make stop-all
```

Reply:
I removed them because in my case, I already started the dependent services so I don't want to start them every time, and also don't want to stop them after finishing the test cases. so, I think making the behavior more unit will be better.